data/widgets/card/title.ui

```diff
@@ -33,7 +33,7 @@
             <property name="use_markup">True</property>
             <property name="wrap">False</property>
             <property name="wrap_mode">word-char</property>
-            <property name="max_width_chars">25</property>
+            <property name="max_width_chars">50</property>
```

Comment:
Why is this changed needed? In any case, can you double check that this is not breaking other apps that use title card in composite mode?